python-script/build_workloads.py

```diff
@@ -21,7 +21,7 @@
 if __name__ == '__main__':
     for dataset in datasets:
         path = os.path.join(source, dataset)
-        gt_path = os.path.join(path, f'{dataset}_benchmark_groundtruth_50000.ivecs')
+        gt_path = os.path.join(path, f'{dataset}_benchmark_groundtruth_recall0.98.ivecs')
```

Comment:
Why hard code here?

Reply:
Temporary code written for testing; it will be deleted.